lib/pure/json.nim

```diff
@@ -818,6 +818,25 @@ proc `%`*(o: enum): JsonNode =
   ## string. Creates a new ``JString JsonNode``.
   result = %($o)
 
+proc `%`*[T](t :TableRef[string,T]) : JsonNode =
```

Comment:
can you fix the code style? it should be like `t: TableRef[string, T]): JsonNode`

Reply:
and also in other places :)

Reply:
sure, is there any tool for style fix?